src/core/stompest/protocol/broker.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+
+class Broker(object):
+    PROTOCOL_TCP = 'tcp'
+    PROTOCOL_SSL = 'ssl'
+    PROTOCOLS = {PROTOCOL_TCP, PROTOCOL_SSL}
```

Comment:
Use `set(...)` not `{...}` because `stompest` needs to remain backward compatible to Python 2.6.